### ISSUE TITLE
path.normalize issues

### ISSUE THREAD

**Issue by akaStiX** [premake/premake-core#316]
It has issues again :(
There is a convention in Premake, that all paths should use **/** for all separators.
Apparently that doesn't work on Windows, when you try to execute some os commands like del\ren, etc.
Fixing this will be trivial, I just cannot do the pull request at the moment, so I'd like to get the opinions first

**Comment by tvandijck** [comment]
well, that's why there is the path.translate method, to make the path windows compatible.
path.normalize always gives paths with '/'... that's not really an issue with path.normalize though, that's just what it does..

just do "path.translate(path.normalize(...))" for os command arguments.

**Comment by akaStiX** [comment]
Well, than this is even simpler.
This is how os.execute looks like
```lua
	premake.override(os, "execute", function(base, cmd)
		cmd = path.normalize(cmd)
		cmd = os.translateCommands(cmd)
		return base(cmd)
	end)
```
it does normalization, but forgets about translation :(

**Comment by starkos** [comment]
We can't just blindly translate the command though, because DOS commands use the forward slash to signify a switch, i.e. `dir /b`.

Suggestions welcome!

**Comment by akaStiX** [comment]
well... Than we should get rid of path.normalize in os.execute at all...

**Comment by starkos** [comment]
Perhaps. The way `path.normalize()` used to work it would clean up the command but not any arguments. Perhaps that has been broken by the recent changes though.

**Comment by akaStiX** [comment]
Some time ago I've fixed path.normalize. Before that it was totally broken and it messed up the entire command

**Comment by akaStiX** [comment]
@starkos  can you take a look on my pull request that solves this issue? https://github.com/premake/premake-core/pull/320

**Comment by samsinsane** [comment]
I believe this issue has been resolved by #570.
